minecraft_servers/voided/manifest.json

```diff
@@ -0,0 +1,36 @@
+{
+  "server_name": "voided",
+  "nice_name": "Voided Network",
+  "direct_ip": "voided.gg",
+  "server_wildcards": ["%.voided.gg"],
+  "supported_languages": ["en"],
+  "social": {
+    "web": "https://voided.gg",
+    "web_shop": "https://store.voided.gg",
```

Comment:
Returns a `DNS_PROBE_FINISHED_NXDOMAIN` error

Reply:
Still not available